scan_dynamoDB/scan_dynamo.js

```diff
@@ -53,14 +53,16 @@ const scanTable = async () => {
     let items;
     do{
       items = await dynamoDB.scan(params).promise();
-      items.Items.forEach((item) => console.log(JSON.stringify(AWS.DynamoDB.Converter.marshall(item))));
+      items.Items.forEach((item) => scanResults.push(AWS.DynamoDB.Converter.marshall(item)));
+      
       params.ExclusiveStartKey = items.LastEvaluatedKey;
     } while(typeof items.LastEvaluatedKey !== "undefined");
     
     return scanResults
 }
 
 scanTable()
-.then((rows) => {
-    console.log('done')
+.then((results) => {
+  fs.writeFileSync(tableName+'_'+new Date().toISOString()+'.json', JSON.stringify(results))
```

Comment:
@ViVeniero potresti aggiornare il README specificando che il risultato viene scritto in un file che ha un certo formato ?